comps/comps-foreman-el8.xml

```diff
@@ -249,7 +247,6 @@
       <packagereq type="default">rubygem-ovirt-engine-sdk</packagereq>
       <packagereq type="default">rubygem-paint</packagereq>
       <packagereq type="default">rubygem-parallel</packagereq>
-      <packagereq type="default">rubygem-passenger</packagereq>
```

Comment:
there is also `rubygem-passenger-doc` in this comps.xml that can be removed.

Reply:
Updated